README.rst

```diff
@@ -1274,7 +1274,7 @@ Constraints and Rules
 ^^^^^^^^^^^^^^^^^^^^^
 
 -  Must be applied on a property field.
--  :code:`value` must be empty.
+-  :code:`value` must not appear in the filter (shown here) or be an empty list.
```

Comment:
```suggestion
-  :code:`value` must either not appear in the filter (shown in the example) or be an empty list.
```
Similar change to the entry below.

Reply:
Should we remove `value` from the examples as well? Seems superfluous.

Reply:
It seems like it is removed — see changes above an below in the `Example use` sections. Are there other examples you were thinking of?